exercises/matrix/src/test/java/MatrixTest.java

```diff
@@ -10,198 +10,100 @@
-                     + "2 3", new int[] {2, 3}},
-                    {  "0 1 2\n"
-                     + "3 4 5\n"
-                     + "6 7 8", new int[] {6, 7, 8}}
-            });
-        }
-
-        public GetLastRowTest(String matrixAsString, int[] lastRow) {
-            this.matrix = new Matrix(matrixAsString);
-            this.lastRow = lastRow;
-        }
-
-        @Test
-        public void getLastRowTest() {
-            assertArrayEquals(lastRow, matrix.getRow(matrix.getRowsCount() - 1));
-        }
-    }
-
-    @Ignore("Remove to run test")
-    @RunWith(Parameterized.class)
-    public static class GetFirstColumnTest {
-        private Matrix matrix;
-        private int[] firstColumn;
-
-        @Parameterized.Parameters(name = "{index}: checking first column of matrix constructed with string \n\"{0}\".")
-        public static Collection<Object[]> data() {
-            return Arrays.asList(new Object[][]{
-                    {"0", new int[] {0}},
-                    {"0 1", new int[] {0}},
-                    {  "0\n"
-                     + "1", new int[] {0, 1}},
-                    {  "0 1\n"
-                     + "2 3", new int[] {0, 2}},
-                    {  "0 1 2\n"
-                     + "3 4 5\n"
-                     + "6 7 8", new int[] {0, 3, 6}}
-            });
-        }
-
-        public GetFirstColumnTest(String matrixAsString, int[] firstColumn) {
-            this.matrix = new Matrix(matrixAsString);
-            this.firstColumn = firstColumn;
-        }
-
-        @Test
-        public void getFirstColumnTest() {
-            assertArrayEquals(firstColumn, matrix.getColumn(0));
-        }
-    }
-
-
-    @Ignore("Remove to run test")
-    @RunWith(Parameterized.class)
-    public static class GetLastColumnTest {
-        private Matrix matrix;
-        private int[] lastColumn;
-
-        @Parameterized.Parameters(name = "{index}: checking last column of matrix constructed with string \n\"{0}\".")
-        public static Collection<Object[]> data() {
-            return Arrays.asList(new Object[][]{
-                    {"0", new int[] {0}},
-                    {"0 1", new int[] {1}},
-                    {  "0\n"
-                     + "1", new int[] {0, 1}},
-                    {  "0 1\n"
-                     + "2 3", new int[] {1, 3}},
-                    {  "0 1 2\n"
-                     + "3 4 5\n"
-                     + "6 7 8", new int[] {2, 5, 8}}
-            });
-        }
-
-        public GetLastColumnTest(String matrixAsString, int[] lastColumn) {
-            this.matrix = new Matrix(matrixAsString);
-            this.lastColumn = lastColumn;
-        }
-
-        @Test
-        public void getLastColumnTest() {
-            assertArrayEquals(lastColumn, matrix.getColumn(matrix.getColumnsCount() - 1));
-        }
-    }
+	
+	@Test
+	public void ExtractRowFromOneNumberMatrixTest() {
+		String matrixAsString = "1";
+		int rowIndex = 0;
+		int[] row = {1};
```

Comment:
A very minor thing, but I think if this is name `expectedRow` then it makes the test a bit easier to read 🙂